02-microarray/pathway-analysis_microarray_01_ora.Rmd

```diff
@@ -170,11 +175,15 @@ library(magrittr)
 
 ## Import data
 
-We will read in the differential expression results we will download from online.
+For ORA, we only need a list of gene IDs as our input, so this example can work for any situations where you have gene list and want to know more about what biological pathways it shares genes with.
+
+For this example, we will read in the differential expression results we will download from online.
 These results are from a zebrafish microarray experiment we used for [differential expression analysis for two groups](https://alexslemonade.github.io/refinebio-examples/02-microarray/differential-expression_microarray_02_2-groups.html) using [`limma`](https://bioconductor.org/packages/release/bioc/html/limma.html) [@Ritchie2015].
 The table contains Ensembl gene IDs, log fold-changes for each group, and adjusted p-values (FDR in this case).
 We can identify differentially regulated genes by filtering these results and use this list as input to ORA.
 
+Generally, the dataset we are using here would be a good candidate for [GSEA](https://alexslemonade.github.io/refinebio-examples/02-microarray/pathway-analysis_microarray_02_gsea.html) since it has gene-level rankings, but here we are showing you how to use ORA since it can be useful in a number of other situations. 
```

Comment:
I don't feel like we need to belabor this? If we want to say it again, I would probably put it toward the end with conclusions, rather than at this stage where we are focused on getting the data arranged for analysis.

Reply:
I deleted this mention